admin/client/dist/js/MemberDatetimeOptionsetField.js

```diff
@@ -32,9 +32,9 @@
 
 		$('.memberdatetimeoptionset .toggle').entwine({
 			onclick: function onclick(e) {
-				jQuery(this).closest('.form__field-description').parent().find('.toggle-content').toggle();
+				jQuery(this).closest('.description').parent().find('.toggle-content').toggle();
```

Comment:
Sorry, dist files are automatically built and isn't recommended to be manually modified, when #5955 gets merged in then the regression will no longer be there. It'll be difficult to keep track of a dist file change, for example if this had been minified.